cli.js

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/env node
```

Comment:
This is the entry point to the tool - you should update `package.json` if the entry point to the cli changes. I do want to keep this file, since we want to add support for different commands in the future - `esprint check` or `esprint stop`, similar to flow